src/Invictus.Testing.Tests.Integration/TestConfig.cs

```diff
@@ -39,6 +40,23 @@ public static TestConfig Create()
             return new TestConfig(configuration);
         }
 
+        /// <summary>
+        /// Gets the Azure Logic App definition to test update interactions.
+        /// </summary>
+        public string GetLogicAppDefinition()
+        {
+            var fileName = "rcv-trigger-http.json";
```

Comment:
Perhaps change this to an `private` method which is being called from a `public` method called "`GetLogicAppTriggerUpdateDefinition`", which provides the correct filename to the `private` method.

Allows to easily re-use this method if other definitions would have to be loaded in the future for other test-cases.

Reply:
Done.